Formula/felinks.rb

```diff
@@ -0,0 +1,127 @@
+
+    # Allow screen-88color to have 88 colors, just like xterm-88color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-88color.colors", NULL) = COLOR_MODE_88;',
+      'get_opt_int("terminal.xterm-88color.colors", NULL) = COLOR_MODE_88;
+       get_opt_int("terminal.screen-88color.colors", NULL) = COLOR_MODE_88;'
+
+    # Allow screen-88color to underline, just like xterm-88color
+    inreplace "src/config/options.c",
+      'get_opt_bool("terminal.xterm-88color.underline", NULL) = 1;',
+      'get_opt_bool("terminal.xterm-88color.underline", NULL) = 1;
+       get_opt_bool("terminal.screen-88color.underline", NULL) = 1;'
+
+    # Allow screen-color to be a VT100 variant, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-color.type", NULL) = TERM_VT100;',
+      'get_opt_int("terminal.xterm-color.type", NULL) = TERM_VT100;
+       get_opt_int("terminal.screen-color.type", NULL) = TERM_VT100;'
+
+    # Allow screen-color to have colors, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-color.colors", NULL) = COLOR_MODE_16;',
+      'get_opt_int("terminal.xterm-color.colors", NULL) = COLOR_MODE_16;
+       get_opt_int("terminal.screen-color.colors", NULL) = COLOR_MODE_16;'
+
+    # Allow screen-color to underline, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_bool("terminal.xterm-color.underline", NULL) = 1;',
+      'get_opt_bool("terminal.xterm-color.underline", NULL) = 1;
+       get_opt_bool("terminal.screen-color.underline", NULL) = 1;'
+
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}",
+                          "--disable-nls",
+                          "--enable-256-colors",
+                          "--enable-88-colors",
+                          "--enable-bittorrent",
+                          "--enable-cgi",
+                          "--enable-exmode",
+                          "--enable-finger",
+                          "--enable-gemini",
+                          "--enable-gopher",
+                          "--enable-html-highlight",
+                          "--enable-nntp",
+                          "--enable-true-color",
+                          "--with-brotli",
+                          "--with-openssl",
+                          "--with-tre",
+                          "--without-gnutls",
+                          "--without-perl",
+                          "--without-spidermonkey",
+                          "--without-xterm"
+    system "make"
+    system "make", "install"
+
+    # Rename binary and man pages to avoid clashing with elinks
+    mv bin/"elinks", bin/"felinks"
+    mv man/"man1/elinks.1", man/"man1/felinks.1"
+    mv man/"man5/elinks.conf.5", man/"man5/felinks.conf.5"
+    mv man/"man5/elinkskeys.5", man/"man5/felinkskeys.5"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/felinks --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106082/commits/a4fd7e98405e6e2a98e2cc0f7af88c550073e028.